src/ripple/consensus/Consensus.h

```diff
@@ -88,6 +91,8 @@ checkConsensus(
     std::chrono::milliseconds currentAgreeTime,
     ConsensusParms const & parms,
     bool proposing,
+    std::size_t const laggards,
```

Comment:
nit: having two params of the same type is easy to mix up at the call-site. 

Best solution is probably to adopt more tagged integers.